saas/managers/metrics.py

```diff
@@ -246,14 +230,14 @@ def aggregate_monthly_transactions(organization, account,
     if not account_title:
         account_title = str(account)
     interval = organization.natural_interval
-    customers, account_totals = aggregate_monthly_churn(organization, account,
-        interval, from_date=from_date, tz=tz, orig=orig, dest=dest)
+    customers, account_totals = _aggregate_transactions_change_by_period(organization, account,
+        interval, date_periods=date_periods, orig=orig, dest=dest)
     churned_custs, total_custs, new_custs = customers
     churned_account, total_account, new_account = account_totals
     net_new_custs = []
     cust_churn_percent = []
     last_nb_total_custs = 0
-    for index in range(0, 12):
+    for index in range(0, len(date_periods) - 2):
```

Comment:
It seems `len(date_periods) - 1` would be correct with 13 date points / 12 periods.